shell/app/common/components/tree/tree.tsx

```diff
@@ -22,6 +22,8 @@ import { EditCategory } from './edit-category';
 import { findTargetNode, getIcon, isAncestor, walkTree } from './utils';
 import { WithAuth } from 'user/common';
 
+import './tree.scss';
```

Comment:
Why this file is under `.file-tree-container` className?

Reply:
Because the <Tree/> in the file has 'file-tree-container' className, should I change the className of <Tree/> to 'tree-category'.